### PR TITLE
Adding new candidate for MyBatis Migrations as migrate

### PR DESCRIPTION
**Pull request by xshyamx**
Adding new candidate as `migrate` for [MyBatis Migrations](https://mybatis.org/migrations/) a tool that helps manage database migrations via sql scripts and a command line

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/MyBatisMigrations.scala [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+class MyBatisMigrations {
+  @ChangeSet(
+    order = "001",
+    id = "001-add_mybatis_migrations_candidate",
+    author = "xshyamx"
+  )
+  def migration001(implicit db: MongoDatabase) = {
+    Candidate(
+      candidate = "migrate",
+      name = "Apache MyBatis Migrations",
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/MyBatisMigrations.scala:16]: Let's keep the name short. `MyBatis` should do.

File: src/main/scala/io/sdkman/changelogs/MyBatisMigrations.scala [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      "3.3.1",
+      "3.3.0",
+      "3.1.1"
+    ).map(
+        version =>
+          Version(
+            candidate = "migrate",
+            version = version,
+            url =
+              s"https://github.com/mybatis/migrations/releases/download/mybatis-migrations-$version/mybatis-migrations-$version-bundle.zip"
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/MyBatisMigrations.scala:48]: The releases are not great to depend on as they are removed after some time. We usually depend on the archives URLs.

File: src/main/scala/io/sdkman/changelogs/MyBatisMigrations.scala [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      "3.2.1",
+      "3.2.0",
+      "3.1.0"
+    ).map(
+        version =>
+          Version(
+            candidate = "migrate",
+            version = version,
+            url =
+              s"https://github.com/mybatis/migrations/releases/download/mybatis-migrations-$version/mybatis-migrations-$version.zip"
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/MyBatisMigrations.scala:72]: Same as other URL.

File: src/main/scala/io/sdkman/changelogs/MyBatisMigrations.scala [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+@ChangeLog(order = "062")
+class MyBatisMigrations {
+  @ChangeSet(
+    order = "001",
+    id = "001-add_mybatis_migrations_candidate",
+    author = "xshyamx"
+  )
+  def migration001(implicit db: MongoDatabase) = {
+    Candidate(
+      candidate = "migrate",
```
[the remark this answers is not in the document]
Comment (xshyamx) [reply src/main/scala/io/sdkman/changelogs/MyBatisMigrations.scala:15]: The installed sdk will have `migrate` & `migrate.cmd` in the `bin` folder. Hence chose migrate. So, that candidate name reflects the final command that will be installed.

File: src/main/scala/io/sdkman/changelogs/MyBatisMigrations.scala (hunk 2 above)
Comment (xshyamx) [reply src/main/scala/io/sdkman/changelogs/MyBatisMigrations.scala:48]: Will change. Initially, I only saw the jar files being published to maven repo but, looking at the version directory the bundle files are also present.

Comment (xshyamx) [reply src/main/scala/io/sdkman/changelogs/MyBatisMigrations.scala:48]: Sorry the https://archive.apache.org/dist/ does not contain MyBatis Migrations only the maven repository does

**Pull request merged**